cmd/mdatagen/metadata-schema.yaml

```diff
@@ -1,6 +1,17 @@
-# Required: type of the receiver.
+# Required: The type of the component.
 type:
 
+# Required: A high-level view of the development status and use of this component
+status:
+  # Required: The class of the component (For example receiver)
+  class: <receiver|processor|exporter|cmd|connector|extension>
+  # Required: The stability of the component - See https://github.com/open-telemetry/opentelemetry-collector#stability-levels
+  stability: <in development|alpha|beta|stable|deprecated|unmaintained>
```

Comment:
Same as for class

Reply:
Stability levels are defined here: https://github.com/open-telemetry/opentelemetry-collector#stability-levels

Reply:
What I meant here was whether we wanted to include this list of possible values in this documentation (Such as receiver, processor, etc for class). The stability levels are rather unlikely to change but the classes are more prone to change.